Source/MonoGame.Extended/Maps/Tiled/TiledMap.cs

```diff
@@ -41,6 +40,7 @@ public void Dispose()
         public int Height { get; }
         public int TileWidth { get; }
         public int TileHeight { get; }
+        public List<TiledTileset> Tilesets { get; }
```

Comment:
@weaversam8 Cool. You could even have a local variable for the tile sets and then just have an explicit property getter that returns an `IReadOnlyList<TiledTileset>`. Here is a snippet:

```
private List<TiledTileset> _tilesets;

public IReadOnlyList<TiledTileset> Tilesets
{
    get { return _tilesets; }
}
```